light/client.go

```diff
@@ -1183,39 +1059,38 @@ func (c *Client) replacePrimaryProvider() error {
 	c.providerMutex.Lock()
 	defer c.providerMutex.Unlock()
 
-	c.logger.Info("Primary is unavailable. Replacing with the first witness")
 	if len(c.witnesses) <= 1 {
 		return errNoWitnesses{}
 	}
 	c.primary = c.witnesses[0]
 	c.witnesses = c.witnesses[1:]
-	c.logger.Info("New primary", "p", c.primary)
+	c.logger.Info("Replacing primary with the first witness", "new primary", c.primary)
```

Comment:
this won't be logged because of the whitespace ` ` in the key name...

```suggestion
	c.logger.Info("Replacing primary with the first witness", "new_primary", c.primary)
```